users/models.py

```diff
@@ -19,30 +37,73 @@ def follow(self, other):
 
     def unfollow(self, other):
         self.friends.remove(other)
-
+        
+    def follow_ext(self,other):
+        self.ext_following.add(other)
+    
+    def unfollow_ext(self,other):
+        self.ext_following.remove(other)
+        
+    def follow_by_ext(self,other):
+        self.ext_follower.add(other)
+        
+    def unfollow_by_ext(self,other):
+        self.ext_follower.remove(other)
+        
```

Comment:
you could merge these into the follow and unfollow functions by checking the type of other and adding follow/unfollow to ext user. (just a suggestion, I don't care if you do it)